rustic-babel.el

```diff
@@ -73,9 +85,12 @@ should be wrapped in which case we will disable rustfmt."
           (toolchain (cond ((eq toolchain-kw-or-string 'nightly) "+nightly")
                            ((eq toolchain-kw-or-string 'beta) "+beta")
                            ((eq toolchain-kw-or-string 'stable) "+stable")
+                           ((or (null rustic-babel-default-toolchain)
+                                (null (string-blank-p rustic-babel-default-toolchain)))
+                            nil)
```

Comment:
this bit doesn't look quite right to me... `string-blank-p` will return `0` if its an empty string, so `null` will be false.

I'm not sure if this is related to the failing tests, but if `rustic-babel-default-toolchain` _were_ set to `""` in the test environment, we wouldn't hit `nil` here and would fall through.

Reply:
eq 0 now, thank you so much! if you have resources/time to see what's going on with the tests, that would be wonderful as I'm having to max focus on other stuff rn.

Reply:
i will give it a shot, i need to set up all the dependencies locally to run tests, so it may be a bit before i can get to it.